ansible/roles/security/tasks/main.yml

```
@@ -0,0 +1,129 @@
+# Security role tasks
+- name: Reset UFW to defaults
+  ufw:
+    state: reset
+  become: yes
+
+- name: Set UFW default policies
+  ufw:
+    direction: "{{ item.direction }}"
+    policy: "{{ item.policy }}"
+  loop:
+    - { direction: 'incoming', policy: 'deny' }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)